tt1-blocks/block-templates/page-home.html

```diff
@@ -3,7 +3,7 @@
 <!-- wp:group {"tagName":"main", "align":"full"} -->
 <main class="wp-block-group alignfull">
 	<div class="wp-block-group__inner-container">
```

Comment:
Question, were we not meant to remove the inner containers too?

Reply:
I think it's safe to remove yes, the deprecation probably triggers here making sure everything works with or without them but yes, it's better to remove.

Reply:
Thanks, I've removed these in https://github.com/WordPress/theme-experiments/pull/236/commits/af20e47b88ba02f1f970ab700b906cb42561d153.